.github/workflows/_dist.yml

```diff
@@ -12,6 +15,9 @@ jobs:
           # Need this to get version number from last tag
           fetch-depth: 0
 
+      - name: Install pyqt dependencies
+        run: sudo apt-get update && sudo apt install -y libgl1 libxcb-cursor0 libgles2-mesa-dev
```

Comment:
Were you able to confirm whether these dependencies are necessary with offscreen?

Reply:
Fine to leave for now, thankfully the changes are relatively self contained.

The only 'problem' is their duplication, but I suspect any way around this will be more of a mess.